### ISSUE TITLE
cannot find source of now.fortuitous.* packages

### ISSUE THREAD

**Issue by IzzySoft** [Tornaco/Thanox#812]
Just wondering: there are many lines with `import now.fortuitous.<something>` – but I cannot find any `package now.fortuitous.<something>`. A web search didn't bring them up either. Can you please point out where to find those? Thanks!

**Comment by IzzySoft** [comment]
I guess those are the non-FOSS parts then?

**Comment by Tornaco** [comment]
@IzzySoft Yes,  there are some packages used for Liscense validation, recently the pkg name was renamed from github.tornaco.xxx to now.focxxx for some personal reasons.

**Comment by IzzySoft** [comment]
Thanks! Added them to your app's "namespaces" with my config here so I'm not constantly alerted about "unknown libraries" being inside your APKs.

**Comment by IzzySoft** [comment]
So is there any chance to get those missing parts FOSS and publicly accessible? We currently have a double issue here:

* Thanox does not meet our inclusion criteria (due to non-FOSS parts)
* we cannot even check it completely (see: [Reproducible Builds](https://izzyondroid.org/about/security/ReproducibleBuilds/)), as that code is not accessible
* our PR concerning the app metadata (#1351 – opened back in August) is still being ignored, making our maintenance harder

So we are currently at the point where we must consider removing [Thanox from IzzyOnDroid](https://apt.izzysoft.de/packages/github.tornaco.android.thanos), @Tornaco – which would be a real pity. And probably cause quite some people currently using the app to become quite unhappy, if I see the install count:

<img width="1230" height="407" alt="Image" src="https://github.com/user-attachments/assets/061995fb-f106-47f0-95aa-6e5a91460d5a" />
